pkg/prompt/selector.go

```diff
@@ -0,0 +1,429 @@
+package prompt
```

Comment:
Was there any effective change in this file compared to the original one? it is hard to see as the diff considers it as a new file.

Reply:
@ifbyol i've commented the code under deleted selector.go and dev.go which have the "moved" items to the new pkg `prompt`